idr0050-all-bulkmap-config.yml

```diff
@@ -0,0 +1,57 @@
+---
+name: idr0050-springer-cytoskeletalsystems
+version: 1
+
+defaults:
+  # Should the column be processed when creating bulk-annotations (yes/no)
+  include: no
+  # Columns type of the bulk-annotations column
+  type: string
+
+  # If non-empty a string used to separate multiple fields in a column
+  # White space will be stripped
+  split: ;
+  # Should this column be included in the clients (yes/no)
+  includeclient: yes
+  # Should this column be visible in the clients, if no the column should be
+  # hidden in the client but will still be indexed by the searcher (yes/no)
+  visible: yes
+  # Should empty values be omitted from the client display
+  omitempty: yes
+
+columns:
+
+  - name: Characteristics [Organism Part]
+    clientname: Organism Part
+    include: yes
+  - name: Experimental Condition [Nocodazole or DMSO]
+    clientname: Pharmacological Treatment
+    include: yes
+  - name: Experimental Condition [stretch duration]
+    clientname: Stretch Duration
+    include: yes
+
+  ######################################################################
+  # mapr groups
+  ######################################################################
+
+  - group:
+      namespace: openmicroscopy.org/mapr/cell_line
+      columns:
+        - name: Characteristics [Cell Line]
+          clientname: Cell Line
+          include: yes
+
+  - group:
+      namespace: openmicroscopy.org/mapr/organism
+      columns:
+        - name: Characteristics [Organism]
+          clientname: Organism
+          include: yes
+
+advanced:
+  ignore_missing_primary_key: yes
+  primary_group_keys:
+    - namespace: openmicroscopy.org/mapr/organism
+      keys:
+        - Organism
```

Comment:
Since `Cell line` is a primary group key, it needs to be added to this list (as in https://github.com/IDR/idr0041-cai-mitoticatlas/blob/master/experimentA/idr0041-experimentA-bulkmap-config.yml#L78) to prevent the duplication of map annotations.